Makefile

```diff
@@ -16,11 +16,7 @@ pip_requirements:
 
 django_migrations:
 	$(PYTHON) manage.py base_specify_migration
-	$(PYTHON) manage.py migrate specify
-	$(PYTHON) manage.py migrate notifications
-	$(PYTHON) manage.py migrate workbench
-	$(PYTHON) manage.py migrate accounts
-	$(PYTHON) manage.py migrate permissions
+	$(PYTHON) manage.py migrate
```

Comment:
Should we remove the call to run migrations in the `docker-entrypoint.sh` file instead so this is not redundant?